production/helm/loki/src/rules.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
```

Comment:
Where are these rules used?

Reply:
recording rules used in our dashboards, but they're not actually wired up to work yet